projects/commudle-admin/src/app/components/home/communities/home-community-card/home-community-card.component.scss

```diff
@@ -48,3 +48,55 @@
     }
   }
 }
+
+
+@media only screen and (max-width: 768px){
+  nb-card {
+    width: 100%;
+    height: 120px !important;
```

Comment:
Why do we need to use !important here?
It should automatically work on this screen size